dnf-plugins-core.spec

```diff
@@ -86,6 +86,52 @@ reposync commands. Additionally provides generate_completion_cache and noroot
 passive plugins.
 %endif
 
+%package -n dnf-utils
+Conflicts:      yum-utils < 1.1.32
+Requires:       dnf >= %{dnf_lowest_compatible}
+Requires:       %{name} = %{version}-%{release}
+%if %{with python3}
+Requires:       python3-dnf-utils = %{version}-%{release}
+Requires:       python3-dnf-plugins-extras-repoclosure
+%else
+Requires:       python2-dnf-utils = %{version}-%{release}
+Requires:       python2-dnf-plugins-extras-repoclosure
+%endif
+Summary:        Yum-utils CLI compatibility layer
+
+%description -n dnf-utils
+As a Yum-utils CLI compatibility layer, supplies in /usr/bin/ builddep, repoquery, repoclosure,
+download, debuginfo-install, and config-manager and redirecting to DNF.
+
+%package -n python2-dnf-utils
```

Comment:
I would prefer to not pollute repomd with such useless packages as they don't provide any python modules.